index.js

```diff
@@ -278,7 +276,7 @@ RingPop.prototype.bootstrap = function bootstrap(bootstrapFile, callback) {
         self.isReady = true;
         self.emit('ready');
 
-        if (callback) callback();
+        if (callback) callback(err, nodesJoined);
```

Comment:
nitpick: doing `callback(null, nodesJoined)` makes it slightly clearer then we early returned for all the error cases